content/docs/1.10.0/advanced-resources/cluster-restore/rancher-cluster-restore.md

```diff
@@ -16,7 +16,7 @@ This doc describes what users need to do after restoring the cluster with a Ranc
 - Detect the invalid or out-of-sync replicas as long as the related volume still contains a valid replica after the restore.
 
 ## Behaviors & Requirement of Rancher restore
-- According to [the Rancher restore article](https://rancher.com/blog/2018/2018-05-30-recover-rancher-kubernetes-cluster-from-backup/), you have to restart the Kubernetes components on all nodes. Otherwise, there will be tons of resource update conflicts in Longhorn.
+- You have to restart the Kubernetes components on all nodes. Otherwise, there will be tons of resource update conflicts in Longhorn.
```

Comment:
```suggestion
According to [the Rancher restore article](https://ranchermanager.docs.rancher.com/how-to-guides/new-user-guides/backup-restore-and-disaster-recovery/restore-rancher-launched-kubernetes-clusters-from-backup), you have to restart the Kubernetes components on all nodes. Otherwise, there will be tons of resource update conflicts in Longhorn.
```

Could we replace the link with this one [Restoring a Cluster from Backup](https://ranchermanager.docs.rancher.com/how-to-guides/new-user-guides/backup-restore-and-disaster-recovery/restore-rancher-launched-kubernetes-clusters-from-backup)?
And the original article is still kept here [Recover Rancher Kubernetes cluster from a Backup](https://www.rancher.cn/blog/2018/2018-05-30-recover-rancher-kubernetes-cluster-from-backup/)